content/webapp/components/LinkLabels/LinkLabels.tsx

```diff
@@ -19,23 +21,22 @@ type Props = {
 
 type LinkOrSpanSpaceAttrs = {
   url?: string;
-  addBorder: boolean;
+  $addBorder: boolean;
 };
 
 const ItemText = styled(Space).attrs<LinkOrSpanSpaceAttrs>(props => ({
   as: props.url ? 'a' : 'span',
   href: props.url || undefined,
-  h: {
+  className: font('intr', 5),
+  $h: {
     size: 's',
-    properties: [
-      'margin-right',
-      props.addBorder ? 'padding-left' : undefined,
-    ].filter(Boolean),
+    properties: ['margin-right', props.$addBorder ? 'padding-left' : ''].filter(
+      Boolean
+    ) as HorizontalSpaceProperty[],
```

Comment:
type was erroring as it was convinced "undefined" was an option, even though we're filtering.